code/controllers/subsystems/initialization/misc_late.dm

```diff
@@ -36,6 +36,10 @@
 
 	populate_code_phrases()
 
+	// this covers mapped in drone fabs
+	for(var/obj/machinery/drone_fabricator/DF in SSmachinery.all_machines)
+		DF.enable_drone_spawn()
+
```

Comment:
Unsure about that, as this would prevent mappers to intentionally map in disabled drone spawners